core-web/libs/edit-content/src/lib/edit-content.routes.ts

```diff
@@ -13,7 +13,11 @@ export const DotEditContentRoutes: Route[] = [
                 title: 'Create Content',
                 component: EditContentLayoutComponent
             },
-            { path: ':id', title: 'Edit Content', component: EditContentLayoutComponent }
+            {
+                path: ':id',
+                title: 'Edit Content',
+                component: EditContentLayoutComponent
```

Comment:
@oidacra I suggest that we could load this component in lazy load mode, like this: `loadComponent: () => import('./feature/edit-content/edit-content.layout.component')`

Reply:
`edit-content`  already is lazy-load
```
 {
        canActivate: [editContentGuard],
        path: 'content',
        loadChildren: () => import('@dotcms/edit-content').then((m) => m.DotEditContentRoutes)
    },
```